magpie/ui/home/static/style.css

```diff
@@ -1268,3 +1268,9 @@ div.input-container {
     height: 100%;
     -webkit-box-sizing: border-box;
 }
+
+textarea {
+    resize: both;
+    font-size: 13.3333px;
+    font-family: arial, sans-serif;
```

Comment:
Are the font parts needed? 
I think there are generic ones applied for the whole CSS.

It it is indeed necessary, you can remove them from here and add `textarea` at the same place where other fonts are defined.

Reply:
It is necessary as you can see on this image, if I remove those font parts : <img src="https://user-images.githubusercontent.com/36516122/135278614-339b3ff2-2030-4f21-b402-415e6a50dc89.png" width="400">

I checked and if not defined, it uses textarea's user agent stylesheet, which uses `monospace` font-family instead of `arial`.
I put those font settings to reproduce the settings used in the other `input[type="text"]` found on the same page, but by looking at those other text inputs, I see that they also use the default user agent stylesheet. It's just that the default user agent stylesheet settings are different between `input[type="text"]` and `textarea`. So, I will keep those font settings in the file, but wondering if it is okay to leave the `input[type="text"]` with the default settings. Let me know if I should adjust anything!

Also, not really sure, I don't really see a specific place where the other fonts are defined, there are a few definitions scattered in the file, but not really in one specific place. I think I will just move it to the more generic section near the top of the file.

Reply:
It's ok to apply the font specifically for `textarea`. 
I would like to have one place where they are all defined. 

So somewhere at the top under the `html` and `body` part, you can define this: 

```css
table,
textarea,
input[type="text"],
.panel-line    /* others??? */ {  
    font-family: arial, sans-serif;
	font-size: inherit;    /* needed? */
}
```

I'm still not sure about the `font-size: 13.3333px;` though. 
It is hard to tell who everything resolves by static analysis will the overrides that occurs in there. 

It should use the same value as the text inputs. 
If its not the case, you can also apply it to the same 'font' style.